ui/lib/core/addon/utils/search-select-has-many.js

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
I prepended the name of this file with `search-select` so the util didn't get lost and would appear while searching for the `search-select` component.

Reply:
smart!

Reply:
💯